com.chartboost.helium/Editor/HeliumIntegrationChecker.cs

```diff
@@ -68,12 +69,22 @@ public static void ReimportExistingHeliumSamples(ICollection<string> existingSam
             AssetDatabase.Refresh();
 
             var allSamples = Sample.FindByPackage(HeliumPackageName, version);
-
+            var sb = new StringBuilder();
+            sb.AppendLine("<color='green'>[Helium Integration Checker] Ad Adapter Reimport Started!</color>");
```

Comment:
Would it be more helpful to use the internal `Log` method here instead of the `AppendLine`s?

Reply:
This part specifically needs to build information based off the content. We want to be able to Log all information in a single message. However, Rich Text, does not work with multi line messages, unless you do something like above. @cb-jpadilla